challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,23 @@
+
+function binarySearch(arr, num) {
+  
+  let beginning = 0;
```

Comment:
i think this could have been written with a much simpler for loop. but good job getting it done